vignettes/epidatr.Rmd

```diff
@@ -11,379 +11,224 @@ vignette: >
-
-```{r, eval = FALSE}
-pub_covid_hosp_facility_lookup(city = "southlake")
-pub_covid_hosp_facility_lookup(state = "WY")
-# A non-example (there is no city called New York in Wyoming)
-pub_covid_hosp_facility_lookup(state = "WY", city = "New York")
-```
-
-#### COVID-19 Hospitalization by Facility
+## Getting versioned data
 
-API docs: <https://cmu-delphi.github.io/delphi-epidata/api/covid_hosp_facility.html>
+The Epidata API stores a historical record of all data, including corrections
+and updates, which is particularly useful for accurately backtesting
+forecasting models. To fetch versioned data, we can use the `as_of`
+argument.
 
 ```{r, eval = FALSE}
-pub_covid_hosp_facility(
-  hospital_pks = "100075",
-  collection_weeks = epirange(20200101, 20200501)
+# Obtain the smoothed covid-like illness (CLI) signal from the COVID-19
+# Trends and Impact survey for Pennsylvania as it was on 2021-06-01
+pub_covidcast(
+  source = "fb-survey",
+  signals = "smoothed_cli",
+  geo_type = "state",
+  time_type = "day",
+  geo_values = "pa",
+  time_values = epirange(20210105, 20210410),
+  as_of = "2021-06-01"
 )
 ```
 
-#### COVID-19 Hospitalization by State
-
-API docs: <https://cmu-delphi.github.io/delphi-epidata/api/covid_hosp.html>
-
-```{r, eval = FALSE}
-pub_covid_hosp_state_timeseries(states = "MA", dates = "20200510")
-```
-
-### Flu Endpoints
-
-#### Delphi's ILINet forecasts
-
-API docs: <https://cmu-delphi.github.io/delphi-epidata/api/delphi.html>
-
-```{r, eval = FALSE}
-del <- pub_delphi(system = "ec", epiweek = 201501)
-names(del[[1L]]$forecast)
-```
-
-#### FluSurv hospitalization data
-
-API docs: <https://cmu-delphi.github.io/delphi-epidata/api/flusurv.html>
-
-```{r, eval = FALSE}
-pub_flusurv(locations = "ca", epiweeks = 202001)
-```
-
-#### Fluview data
-
-API docs: <https://cmu-delphi.github.io/delphi-epidata/api/fluview.html>
-
-```{r, eval = FALSE}
-pub_fluview(regions = "nat", epiweeks = epirange(201201, 202001))
-```
-
-#### Fluview virological data from clinical labs
-
-API docs: <https://cmu-delphi.github.io/delphi-epidata/api/fluview_clinical.html>
-
-```{r, eval = FALSE}
-pub_fluview_clinical(regions = "nat", epiweeks = epirange(201601, 201701))
-```
-
-#### Fluview metadata
-
-API docs: <https://cmu-delphi.github.io/delphi-epidata/api/fluview_meta.html>
-
-```{r, eval = FALSE}
-pub_fluview_meta()
-```
-
-#### Google Flu Trends data
-
-API docs: <https://cmu-delphi.github.io/delphi-epidata/api/gft.html>
-
-```{r, eval = FALSE}
-pub_gft(locations = "hhs1", epiweeks = epirange(201201, 202001))
-```
-
-#### ECDC ILI
+See `vignette("versioned-data")` for details and more ways to specify versioned data.
 
-API docs: <https://cmu-delphi.github.io/delphi-epidata/api/ecdc_ili.html>
 
-```{r, eval = FALSE}
-pub_ecdc_ili(regions = "Armenia", epiweeks = 201840)
-```
+## Plotting
 
-#### KCDC ILI
+Because the output data is in a standard `tibble` format, we can easily plot
+it using `ggplot2`:
```

Comment:
Should we suggest any particular plotting libraries, especially for more complex plots like choropleths?

Reply:
A choropleth example would be neat if there's a package we could use, but it's pretty low priority imo.